src/app/products/web3storage/page.tsx

```diff
@@ -0,0 +1,78 @@
+import { Header } from "@/components/nav"
+import { Metadata } from "next"
+import Image from "next/image"
+import Ring from '@/../public/images/illustrations/ring.png'
+
+export const metadata: Metadata = {
+  title: 'Features'
+}
+
+export default async function FeaturesPage () {
+  return (
+    <div>
+      <div className="bg-grad">
+        <Header />
+        <section className='max-w-6xl mx-auto pt-20 md:pt-28 px-3 pb-8'>
+          <div className="md:flex flex-row-reverse">
+            <div className="md:w-2/3">
+              <h1 className="font-bold text-4xl text-zinc-950">What is web3.storage</h1>
+              <p className="leading-relaxed pt-8 max-w-3xl text-xl text-zinc-950">At the core of web3.storage is a storage service to safely secure and make your data available - giving developers the power of decentralized storage and content addressing via simple client libraries or an HTTP API.</p>
+            </div>
+            <div className="hidden md:block w-1/3">
+              <Image src={Ring} alt='Ring' className="rotate-45" />
+            </div>
+          </div>
+        </section>
+      </div>
+      <section className="lg:-mt-36 bg-zinc-950 text-zinc-200">
+        <div className='max-w-6xl mx-auto py-24 px-3 md:flex flex-row-reverse items-end'>
+          <div className="md:w-2/3">
+            <p className="text-md leading-relaxed">
+              Content uploaded to web3.storage is stored on our instance of <a href="https://github.com/elastic-ipfs/elastic-ipfs">Elastic IPFS</a>, a cloud-native implementation of IPFS that our team wrote to stay reliable and performant with the scale of uploads we were seeing. Elastic IPFS ensures your data is continuously and quickly available over the IPFS network once your data is uploaded.
+            </p>
+            <p className="text-md leading-relaxed pt-8">
+              Before data is uploaded, it is locally converted into a CAR file, which contains your data transformed into a format that IPFS can consume (a directed acyclic graph, or <a href='https://dag.ipfs.tech/'>DAG</a>). This is very powerful - in this process, a  <a href="https://docs.ipfs.tech/concepts/content-addressing/">content address</a> is locally generated for your data, giving you a guarantee of your data&apos;s fingerprint. Further, you can construct the DAG in useful ways, like making individual files within your folders or key-value pairs in your JSON have their own content addresses.
```

Comment:
Don't need to say "locally" since we already prefixed with "Before data is uploaded".

"Packed" is probably more appropriate since it's a container format.

```suggestion
              Before data is uploaded, it is packed into a CAR file, which contains your data transformed into a format that IPFS can consume (a directed acyclic graph, or <a href='https://dag.ipfs.tech/'>DAG</a>). This is very powerful - in this process, a  <a href="https://docs.ipfs.tech/concepts/content-addressing/">content address</a> is locally generated for your data, giving you a guarantee of your data&apos;s fingerprint. Further, you can construct the DAG in useful ways, like making individual files within your folders or key-value pairs in your JSON have their own content addresses.
```